ajax/libs/zrender/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "zrender",
+  "version": "3.5.2",
+  "filename": "zrender.min.js",
+  "description": "A lightweight canvas library which providing 2d draw for ECharts.",
+  "keywords": [
+    "canvas",
+    "2d",
+    "draw",
+    "echarts"
+  ],
+  "homepage": "https://ecomfe.github.io/zrender/",
+  "authors": [
+    {
+      "name": "Kenner",
+      "email": "kener.linfeng@gmail.com"
+    },
+    {
+      "name": "Yi Shen",
+      "url": "https://github.com/pissang"
+    },
+    {
+      "name": "Shuang Su",
+      "url": "https://github.com/100pah"
+    }
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ecomfe/zrender.git"
+  },
+  "license": "BSD-2-Clause",
```

Comment:
This should be `BSD-3-Clause`.
Ref: https://libraries.io/bower/zrender and https://libraries.io/npm/zrender